contributing_to_docs/contributing.adoc

```diff
@@ -0,0 +1,143 @@
+= Contribute to OpenShift documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about the OpenShift GitHub repository
+
+toc::[]
+
+== Different ways to contribute
+There are a few different ways you can contribute to OpenShift documentation:
+
+* Submit comments at the bottom of each topic (still awaiting implementation)
+* Email the OpenShift documentation team openshift-docs@redhat.com
+* https://github.com/openshift/openshift-docs/issues/new[Create an issue in GitHub] 
+* If you would like to do the work yourself, or if it is a substantial change, then you you should clone the repository, make your changes, and submit a PR
+
+*What happens when you submit a PR?*
+
+If you make a substantial change and submit a PR in GitHub, your PR will be reviewed by our documentation team. If there are any further changes, updates, or corrections required we will inform you in the PR. When the PR has been reviewed and all changes have been accepted by our documentation team, your PR will be merged.
+
+== Repository organization
+Each top directory in the OpenShift documentation repository can include a collection of top level topics, and/or subdirectories that further contain a second level of topics. The exceptions to this rule are directories whose names start with an underscore (like `_builder_lib` and `_javascripts`), which contain the assets used to generate the finished documentation.
+
+Each top level 'topic' directory contains the topics as asciidoc files, any 'subtopic' subdirectories, and an 'images' directory where all images for that topic are stored. 
+
+----
+/
+/topic_dir1
+/subtopic_dir1
+/subtopic_dirN
+/topic_dir/topic1.adoc
+/topic_dir/topicN.adoc
+/topic_dir/subtopic_dir1/topic1.adoc
+/topic_dir/subtopic_dirN/topicN.adoc
+/topic_dir/images
+/topic_dir/images/img1.png
+/topic_dir/images/imgN.png
+...
+/topic_dir2
+----
+
+== Version management
+Most of the content applies to all three OpenShift products: Origin, Online, and Enterprise, so approximately 80% of the content is reused. In many cases, this means that individual topics may need to include or exclude individual paragraphs with respect to a specific OpenShift product. While it is _possible_ to accomplish this solely with Git branches to maintain slightly different versions of a given topic, doing so would make the task of maintaining internal consistency extremely difficult for content contributors.
+
+Git branching is still extremely valuable, and serves the important role of tracking the release versions of documentation for the various OpenShift products.
+
+=== Conditional text between products
+OpenShift documentation uses AsciiDoc's `ifdef/endif` macro to conditionalize and reuse content across the different OpenShift products, down to the single-line level. 
+
```

Comment:
@CowboysFan I want to suggest something for the above sentence, but I'm not sure if I can word it correctly. It's saying that we reuse content across the products, but should it be saying that it's across the product's _documentation_ ?